src/NUnitConsole/nunit3-console/ConsoleRunner.cs

```diff
@@ -31,6 +31,7 @@
 using NUnit.Engine.Extensibility;
 using System.Runtime.InteropServices;
 using System.Text;
+using NUnit.Engine.Internal;
```

Comment:
Since the console does not reference the engine, it seems a bit confusing to have the utility in the `NUnit.Internal.Namespace`. Can it go somewhere else that signals it is used in common?

Reply:
Looks like we still have an NUnit.Common namespace. I can switch it to that. 🙂